service/src/main/java/bio/terra/workspace/service/workspace/flight/DeleteWorkspaceStateStep.java

```diff
@@ -15,22 +15,22 @@
 public class DeleteWorkspaceStateStep implements Step {
```

Comment:
I've been calling this DB state "metadata", not state. You could also say "dbState" I guess.